README.md

```diff
@@ -1,13 +1,14 @@
 # Project Auth
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was about how to build a registration flow, how to handle authentication, both in the frontend and in the backend
+and how to build a frontend and backend at the same time.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We started with building the backend to get an accesstoken per user and added different routes depending if user had accesstoken or not. We had a minor problem by displaying our thougts after the user had logged in but we got help to solve that in our team session. If we had more time we would have focused on displaying better interactions with the user like better error messages etc.
+The user can both create a user and sign in with existing user credentials.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Link to frontend: https://project-auth-mj.netlify.app/
```

Comment:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```

![image](https://user-images.githubusercontent.com/50926159/120888440-98242900-c5f8-11eb-8436-0b17cdd7088b.png)